packages/greedy_infomax/src/nupic/research/frameworks/greedy_infomax/models/bilinear_info.py

```diff
@@ -211,3 +208,133 @@ def __init__(
             )
             for _ in range(self.k_predictions)
         )
+
+
+class _BilinearInfo(nn.Module):
+    """
+    From the Greedy InfoMax paper, a module which estimates the mutual information
+    between two representations through a bilinear model. This is now named
```

Comment:
This docstring should look something like the following to make it clear to the user that it's deprecated:

```
class _BilinearInfo(nn.Module):
    """
    NOTE: This class is deprecated.

    ...
    """
```

Reply:
Makes sense, fixed